### PR TITLE
Added sbrkmerge debug message

### PR DESCRIPTION
**Pull request by keedon**
This is where the error is for mahjongg

### CODE REVIEW

**Comment by elbing** [comment]
This is for avoid kernel crashing, no? or does it fix entire error?

File: sys/src/9/amd64/qmalloc.c [hunk 1]
```diff
@@ -136,7 +136,7 @@ static Pool pimagmem = {
 	.quantum=	32,
 	.alloc=		malloc,
 	.merge=		sbrkmerge,
-	.flags=		0,
+	.flags=		POOL_PARANOIA | POOL_VERBOSITY | POOL_DEBUGGING | POOL_LOGGING,
```
Comment (rminnich) [inline sys/src/9/amd64/qmalloc.c]: This should not be the default, so you're going to have to remove it from this commit.

File: sys/src/9/amd64/qmalloc.c [hunk 2]
```diff
@@ -154,8 +154,11 @@ sbrkmerge(void *x, void *y)
 	uint32_t *lx, *ly;
 
 	lx = x;
-	if(lx[-1] != 0xDeadBeef)
+	if(lx[-1] != 0xDeadBeef){
```
Comment (rminnich) [inline sys/src/9/amd64/qmalloc.c]: This looks fine however.

**Comment by rminnich** [comment]
:-1: 
You need to remove the setting of default attributes. It can't go in in this form. The message is ok however.

**Pull request closed without merging**